android/app/src/main/java/com/arthenica/mobileffmpeg/Config.java

```diff
@@ -84,12 +84,13 @@
 
         /* LOAD NOT-LOADED LIBRARIES ON API < 21 */
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
-            final List<String> externalLibrariesEnabled = getExternalLibraries();
-            if (externalLibrariesEnabled.contains("tesseract") || externalLibrariesEnabled.contains("x265") || externalLibrariesEnabled.contains("snappy") || externalLibrariesEnabled.contains("openh264") || externalLibrariesEnabled.contains("rubberband")) {
-                // libc++_shared.so included only when tesseract or x265 is enabled
+            try {
                 System.loadLibrary("c++_shared");
             }
-            System.loadLibrary("cpufeatures");
+            catch (UnsatisfiedLinkError ex) {
+                Log.i(Config.TAG, "libc++_shared.so fialed to load, try to proceed without it");
```

Comment:
Can we revert this change? We know which libraries depend on `c++`. We even have a wiki page about it. Not using that information and printing an unnecessary log line doesn't make sense to me.

Reply:
We can simply skip the log line, as well. I thought that reducing complex decision here would make maintenance easier: there is only one place in code that decides to package C++ runtime, and the rest follow that decision. This way the decision may be refined. Like, as I found, **openh264** only needs C++ runtime in LTS build.

But if you are uncomfortable with this, I will happily revert this change.

Reply:
So, in the meanwhile, I simply revert this change.